File: apps/custom-2-admin/src/api/customer/CustomerCreateInput.ts
import { AddressWhereUniqueInput } from "../address/AddressWhereUniqueInput";
import { MorCreateNestedManyWithoutCustomersInput } from "./MorCreateNestedManyWithoutCustomersInput";
import { OrderCreateNestedManyWithoutCustomersInput } from "./OrderCreateNestedManyWithoutCustomersInput";

export type CustomerCreateInput = {
  address?: AddressWhereUniqueInput | null;
  email?: string | null;
  firstName?: string | null;
  lastName?: string | null;
  mors?: MorCreateNestedManyWithoutCustomersInput;
  orders?: OrderCreateNestedManyWithoutCustomersInput;
  phone?: string | null;
};

File: apps/order-management/src/APIs/Customer/Dtos/CustomerCreateInput.cs
namespace Customer.APIs.Dtos;

public class CustomerCreateInput
{
    public string id { get; }

    public DateTime createdAt { get; }

    public string updatedAt { get; }

    public string? firstName { get; }

    public string? lastName { get; }

    public string? email { get; }

    public string? phone { get; }

    public List<OrderDto>? orders { get; }

    public AddressDto? address { get; }
}
